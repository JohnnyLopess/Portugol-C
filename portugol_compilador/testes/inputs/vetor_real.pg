programa {
    funcao inicio() {
        real v[3]
        v[0] = 3.14
        v[1] = 2.71
        escreva(v[0])
        escreva(v[1])
    }
}
