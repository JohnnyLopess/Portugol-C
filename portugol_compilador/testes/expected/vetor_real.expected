#include <stdio.h>
#include <string.h>

int main() {
    float v[3];
    v[0] = 3.14;
    v[1] = 2.71;
    printf("%f\n", v[0]);
    printf("%f\n", v[1]);
    return 0;
}
